test/encrypted_schema.test.js

```diff
@@ -144,8 +144,13 @@ describe('encrypted schema declaration', function() {
         }
       }
 
-      // Don't forget to add `Int8` to the type registry
-      mongoose.Schema.Types.Int8 = Int8;
+      this.beforeEach(function() {
+        // Don't forget to add `Int8` to the type registry
+        mongoose.Schema.Types.Int8 = Int8;
+      });
+      this.afterEach(function() {
+        delete mongoose.Schema.Types.Int8;
+      });
```

Comment:
```suggestion
      beforeEach(function() {
        // Don't forget to add `Int8` to the type registry
        mongoose.Schema.Types.Int8 = Int8;
      });
      
      afterEach(function() {
        delete mongoose.Schema.Types.Int8;
      });
```

I haven't seen `this.` before those hooks, is that the same thing?

Reply:
VSCode always defaults to `this.` - they're the same.  mocha just binds the free functions to the context that they're running in, calling the suite-level beforeEach: https://github.com/mochajs/mocha/blob/b720ec1b3ca630a90f80311da391b2a0cdfead4e/lib/mocha.js#L104